src/clincoded/static/components/variant_interpretation_summary/header.js

```diff
@@ -62,7 +63,7 @@ class VariantInterpretationSummaryHeader extends Component {
                             {classification ?
                                 <div>
                                     <dt>Date interpretation saved:</dt>
-                                    <dd className="classificationSaved">{moment(getClassificationSavedDate(classification)).format("YYYY MMM DD, h:mm a")}</dd>
+                                    <dd className="classificationSaved">{moment(lastSavedDate).format("YYYY MMM DD, h:mm a")}</dd>
```

Comment:
What do you think about conditionally rendering this, so that the user doesn't see "Invalid Date"?
```
{lastSavedDate ?
  <dd>.......</dd>
  : null}
```

Reply:
Good suggestion! I'll implement and push the change